### DISCUSSION TITLE
Make Reload command similar to Exit & ExitApp

### DISCUSSION THREAD

**Discussion by kyklish** [mark-wiemer/ahkpp#300] [Ideas]
What do you think? #299

**Comment by mark-wiemer** [comment]
Do other IDEs behave this way? Regardless, it seems like a great candidate for a setting for sure :)

**Comment by kyklish** [comment]
My bad. Got read closely documentation about `Reload` command.

`... the reload command should be followed by whatever actions you want taken in the event of a failure ...`

```ahk
Reload
Sleep 1000 ; If successful, the reload will close this instance during the Sleep, so the line below will never be reached.
MsgBox, 4,, The script could not be reloaded. Would you like to open it for editing?
IfMsgBox, Yes, Edit
return
```

It would be wrong treating `reload` like `exit`.
